test/board_test.rb

```diff
@@ -0,0 +1,22 @@
+require "./lib/board"
+require "./lib/cell"
+require "./lib/ship"
+require "minitest/autorun"
+require "minitest/pride"
+
+class BoardTest < MiniTest::Test
+
+  def test_it_exists_with_attributes
+    board = Board.new
+    assert_instance_of Board, board
+    assert_instance_of Hash, board.cells
+    assert_equal 16, board.cells.length
```

Comment:
I think we should use .size because we are working with a grid?
length implies one "side" to measure. they do the same thing so this is ok for now